### ISSUE TITLE
Sorry, you are not allowed to access this page

### ISSUE THREAD

**Issue by webol** [WordPress/gutenberg#8530]
Installed Gutenberg. 
Clicked Active Gutenberg.
Received a "Sorry, you are not allowed to access this page." rest of page was gray.
I expected the editor to be activated.
Hit the back button and the option to install was there again. I installed the classic editor and everything went smoothly.

Wordpress is a multisite.
Using latest version of Chrome.

**Comment by earnjam** [comment]
Was this on the primary site of the network or one of the subsites? 
Is it a subdomain or subdirectory network?
Are you doing anything special with `sunrise.php` or domain mapping?

**Comment by abwdvm** [comment]
Do you use Cloudflare?

**Comment by designsimply** [comment]
This is the only report of this I've seen here so far which may indicate it's related to your upgrade not going as smoothly as it should have or some other permissions or security problem that's hard to pin down. Following are a few more troubleshooting questions I can think of in case they help surface any relevant and helpful information.

What versions are you testing with? (PHP, WordPress, Gutenberg, OS, and browser)

Do you see anything relevant in your server error logs?

Do you have any security plugins installed?

Do you see any errors in the Network tab of your browser developer tools when the error occurs?

**Comment by webol** [comment]
Let see...
What versions are you testing with?
PHP: 5.4.24
WordPress: WordPress 4.9.8 
Gutenberg: 3.4.0
OS: Window 10
browser: Chrome (latest version)
host: Bluehost

Do you see anything relevant in your server error logs?
I didn't see anything in the logs

Do you have any security plugins installed?
No.

Do you see any errors in the Network tab of your browser developer tools when the error occurs?
I don't know what that is. Sorry.

----

Checking today, I see that is does say that it is Network Activated but the editor looks like the classic editor.

**Comment by earnjam** [comment]
I figured out the issue. Unfortunately it is in the Try Gutenberg callout. 😞 

When prompting to activate, it points multisite users toward network activating. On activation, it then redirects using `wp_redirect( self_admin_url( "admin.php?page=gutenberg" ) )`. In a network context, this doesn't exist. We should have used  `admin_url()`. I've opened a ticket on trac about it here: https://core.trac.wordpress.org/ticket/44747#ticket

@webol the plugin should work fine now if you deactivate classic editor. This was just a one time redirect upon activation that failed.
